src/common/Error/styled.js

```diff
@@ -15,6 +15,7 @@ export const ErrorImg = styled.img`
 
 export const ErrorTitle = styled.h1`
```

Comment:
Wydaje mi się, że jest trochę za wysoko i w projekcie w figmie "and try again" jest w nowej linii, więc może <br> by można było dodać. Komentarz dotyczy całego Errora , nie ErrorTitle ;p